app/scripts/app.config.js

```diff
@@ -30,15 +30,15 @@ angular.module('confRegistrationWebApp')
       },
       vars: {
         development: {
-          apiUrl: 'https://api.stage.eventregistrationtool.com/eventhub-api/rest/',
+          apiUrl: 'http://localhost:8080/eventhub-api/rest/',
           tsysEnvironment: 'staging'
         },
         staging: {
-          apiUrl: 'https://api.stage.eventregistrationtool.com/eventhub-api/rest/',
+          apiUrl: 'http://localhost:8080/eventhub-api/rest/',
           tsysEnvironment: 'production'
         },
         production: {
-          apiUrl: 'https://api.eventregistrationtool.com/eventhub-api/rest/',
+          apiUrl: 'http://localhost:8080/eventhub-api/rest/',
```

Comment:
We can't go to production with this. Changing the `development` one should have been enough if you were testing on localhost. This environment library just does string matching with the domain in the address bar.